react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -334,6 +350,18 @@ export default class JabraService extends VendorImplementation {
         return (type as ErrorType === ErrorType.SDK_USAGE_ERROR && (message as string).includes('call lock'));
     }
 
+    async resetHeadsetState () {
+        if (!this.callControl) {
+            return;
+        }
+
+        await this.callControl.takeCallLock();
```

Comment:
I understand the need of this function to ensure the headset is in complete vanilla state upon connecting but it seems odd to me to take the callLock, release the callLock and then possibly take it again during an incoming call